master/internal/command/tensorboard_manager.go

```diff
@@ -160,6 +159,10 @@ func (t *tensorboardManager) newTensorBoard(
 	commandReq *commandRequest,
 	req tensorboardRequest,
 ) (*command, error) {
+	// Warning! Since certain fields are incompatible with the current model.Experiment,
+	// internally this avoids loading certain parts of the experiment configuration so
+	// we can load their tensorboards still.
+	// TODO(brad): Fix this in the experiment configuration backwards compatibility project.
```

Comment:
non-blocking: TODOs should reference ticket numbers.

Reply:
Created a ticket for this.